components/vf-social-links/vf-social-links.scss

```diff
@@ -80,6 +80,11 @@
   }
 }
 
+.dark-mode {
+  display: inline-block;
+  background-color: set-ui-color(vf-ui-color--black);
+  padding: 0 20px;
+}
 
```

Comment:
Why there is a need to define this class explicitly when it is already defined in Sass including BG color here file https://github.com/visual-framework/vf-core/blob/develop/components/vf-social-links/vf-social-links.scss#L84-L119

Reply:
@sandykadam it's actually not defined there but instead its a selector for the elements having class=dark-mode. It'll apply to the child elements and I had to define .dark-mode class and then apply it to main container so that it shows with black background and child elements are in white.

Reply:
Sorry I didn't get you why exactly it is required?
Below is example on live I added class.

<img width="441" alt="Screenshot 2024-06-21 at 17 09 42" src="https://github.com/visual-framework/vf-core/assets/1227241/2f9b5b63-4a19-4ded-af96-b0395c3ebe02">
<img width="470" alt="Screenshot 2024-06-21 at 17 09 46" src="https://github.com/visual-framework/vf-core/assets/1227241/9ee71e1b-2bf2-4f8d-aa42-2415afa6a086">

Reply:
'dark-mode' selector is working correctly and I didn't add any change there. Since you added it to UL, [class*='dark-mode'] selector made the icons inside white. But that didn't change the UL (parent element) to black(or any other colour). Just for the example I've added the 'dark-mode' class so that the parent shows black. Users can override it in their code. I can remove it if it's better. Please check the screenshot attached.

<img width="854" alt="Screenshot 2024-06-21 at 17 43 51" src="https://github.com/visual-framework/vf-core/assets/122287459/7f461381-766d-4a2c-bdaa-16140cf8b07a">

Reply:
@sandykadam I've deleted the .dark-mode class